app/types/ts-proxies.d.ts

```diff
@@ -0,0 +1,14 @@
+// Import all optional peerDependencies types here to centralize all ts-ignore commands
+// These types will have `any` type at runtime until the appropriate package is installed
+// @ts-ignore
+import type * as ElectronBuilderProxy from "electron-builder";
+// @ts-ignore
+import type * as ElectronBuilderUtilProxy from "builder-util";
```

Comment:
While the other two are included in `devDependencies`, `builder-util` is not included, but still getting imported here. If we go with the approach I explained in the other comment, we must definitely add it to `devDependencies`.

Reply:
"builder-util" is an internal dep of "electron-builder", that's why we get typings even if it's not listed in devDependencies